src/communication.h

```diff
@@ -74,10 +77,32 @@ std::pair<Property, SimpleVariant> processCanMessage(const std::vector<std::uint
         byte2 = msg[4U];
         property = static_cast<Property>(msg[2U]);
     }
-
+    
     const std::uint16_t value{static_cast<std::uint16_t>((byte1 << 8U) | byte2)};
+    
+    #if defined(PROPERTY_DEBUG_H)
+    //decode value with little endian
+    const std::uint16_t value_le{static_cast<std::uint16_t>((byte2 << 8U) | byte1)};
+    // decode receiver CAN ID 
+    std::uint16_t receiver_id = (((msg[0U] & 0xF0) *8)) + (msg[1U] & 0x0F);
```

Comment:
maybe we should create distinct variables for msg[0] and msg[1] since we are going to use them again
```
const auto receiverId{msg[1U]};
const auto readWrite{msg[0U]};
```

Reply:
The first I changed, however receiverId and read write needs to be derived via bitmasks frim msg[1U) and msg[0U]:
`receiver_id = (msg[1U] & 0x0F) | ((msg[0U] & 0xF0) << 8U);`
But if you have a more elgant way, I'm open to it :-)

Reply:
Ah right ... I confused myself ...
you are taking 0x6101 and normalizing it to 0x6001 for both read and write.
This is actually a good hint. Those values follow a pattern and we could just save e.g. 0x6001 and add the read write bit on demand. So please go ahead and I will prepare the other change, so that mapping the values will be more straight forward in the future.

sth. like:
```
struct CanMember {
    const CANID CanId;
    const std::string_view name;
    const std::uint16_t Id[2U];
    bool operator<(const CanMember& other) const { return CanId < other.CanId; }

    std::uint16_t ReadId() const {
        return (Id & (0x01 << 8U));
    }

   std::uint16_t WriteId() const {
       return Id;
   }
};

static const CanMember Kessel    {0x180, "Kessel", {0x3000} };
```
what do you think?